src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            Ball ball = lottery.getRandomBall();
+            System.out.println(ball);
```

Comment:
```suggestion
            System.out.println(lottery.getRandomBall());
```            
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/ec77914f-ec41-4ced-b74e-e738505ff627)